tests/utils/command_line_params.cpp

```diff
@@ -1116,3 +1118,9 @@ void add_quark_smear_option_group(std::shared_ptr<QUDAApp> quda_app)
   opgroup->add_option("--smear-nsteps", smear_n_steps, "Number of smearing steps (default 50)");
   opgroup->add_option("--smear-t0", smear_t0, "Index of the time slice (default -1)");
 }
+
+void add_clover_force_option_group(std::shared_ptr<QUDAApp> quda_app)
+{
+  auto opgroup = quda_app->add_option_group("Clover force", "Options controlling clover force testing");
+  opgroup->add_option("--detratio", detratio, "Test a ratio of determinants. Default is false");
```

Comment:
I think `--determinant-ratio` would be clearer, I won't push on renaming `detratio` everywhere though.

Reply:
45eaa3d85351e8bfd22e51ab22338ab7afac555b